_posts/2024-06-04-update-registration.md

```diff
@@ -0,0 +1,9 @@
+---
+title:  "📢 Update--Registration is live for Our Networks 2024"
+date:   2024-06-04 16:26:10 -0400
+---
+
+Tickets for [#ournetworks](tags/ournetworks) are sliding scale $30-120, suggested $60 to allow us to put on the event: [ournetworks.ca#register](https://ournetworks.ca/#register)
```

Comment:
Is there an example of `tags/[[tag]]` working? It kind of looks like a full path like `https://post.lurk.org/tags/ournetworks` would auto resolve. Not sure what will happen with `tags/`

Reply:
Working on DP/social inbox? Or working in what way? 

`tags/` won't go anywhere
`tags/<tag>` i _should_ go to the #tag page. Reviewed Hometown and Mastodon clients and they all follow that pattern

Reply:
Will a relative link work in the single post view? Like https://post.lurk.org/@orga@ournetworks.ca/112538839276990532

Reply:
ohhhhhhh. tested an no, relative from a post doesn't work :(